Redirects tinygltf to use Drake's vendored nlohmann json.

Reasoning for not upstreaming this patch: Drake-specific build option.

--- tiny_gltf.h
+++ tiny_gltf.h
@@ -1705,7 +1705,7 @@ class TinyGLTF {
 
 #ifndef TINYGLTF_NO_INCLUDE_JSON
 #ifndef TINYGLTF_USE_RAPIDJSON
-#include "json.hpp"
+#include <nlohmann/json.hpp>
 #else
 #ifndef TINYGLTF_NO_INCLUDE_RAPIDJSON
 #include "document.h"
